app/players/elves/elf.py

```diff
@@ -0,0 +1,12 @@
+from app.players.player import Player
+from abc import ABC
+
+
+class Elf(Player, ABC):
```

Comment:
You don’t need ABC inheritance here as your class is not realises any abstract methods As well